jitpack.yml

```diff
@@ -0,0 +1,2 @@
+jdk:
+  - openjdk16
```

Comment:
Okay... does it matter now that scalafix is disabled anyway?

Reply:
OpenJDK16 has better metaspace management, so it may help us delay the problem even futher.
See [here](https://openjdk.java.net/jeps/387)